kotlin/codegen/build.gradle.kts

```diff
@@ -38,26 +38,30 @@ tasks.withType<KotlinCompile>().configureEach {
 val shade: Configuration = configurations.maybeCreate("compileShaded")
 configurations.getByName("compileOnly").extendsFrom(shade)
 dependencies {
-  implementation(project(":moshi"))
-  implementation(kotlin("reflect"))
+  // Use `api` because kapt will not resolve `runtime` dependencies without it, only `compile`
```

Comment:
```suggestion
  // Use `api` because kapt will not resolve `runtime` dependencies without it, only `compile`
  // https://youtrack.jetbrains.com/issue/KT-41702
```